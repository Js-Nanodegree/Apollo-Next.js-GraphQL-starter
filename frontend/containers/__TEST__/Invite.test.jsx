import renderer from 'react-test-renderer';
import InviteContainer from '../Invite';

describe('Invite container', () => {
  it('should match snap shot', () => {
    const tree = renderer.create(<InviteContainer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
