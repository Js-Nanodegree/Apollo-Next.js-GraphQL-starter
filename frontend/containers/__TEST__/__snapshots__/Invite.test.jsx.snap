// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invite container should match snap shot 1`] = `
<div
  className="qepopj-0 gBRnsH"
>
  InviteContainer
</div>
`;
